segments/export/prometheus/prometheus.go

```diff
@@ -23,17 +23,20 @@ import (
 	"strconv"
 	"strings"
 	"sync"
+	"time"
 
 	"github.com/bwNetFlow/flowpipeline/pb"
 	"github.com/bwNetFlow/flowpipeline/segments"
+	"github.com/go-co-op/gocron/v2"
 )
 
 type Prometheus struct {
 	segments.BaseSegment
-	Endpoint     string   // optional, default value is ":8080"
-	MetricsPath  string   // optional, default is "/metrics"
-	FlowdataPath string   // optional, default is "/flowdata"
-	Labels       []string // optional, list of labels to be exported
+	Endpoint       string         // optional, default value is ":8080"
+	MetricsPath    string         // optional, default is "/metrics"
+	FlowdataPath   string         // optional, default is "/flowdata"
+	Labels         []string       // optional, list of labels to be exported
+	VacuumInterval *time.Duration // optional, intervall in which counters should be reset
```

Comment:
We should add some emphasis and warning that vacuuming might happen at an inconvenient time which leads to some data loss (at most about one scrape interval's worth).

I believe it might be possible to hook the next `Gather` operation and only vacuum then, which would prevent most of the possible data loss 🤔